src/utils/i18n.js

```diff
@@ -12,14 +12,14 @@ const format = (str, ...rest) => {
 
     var s = str.toString()
     while(s.length > 0){
-        var m = s.match(/\{(?!\{)([\w\d]+)\}(?!\})/)
+        var m = s.match(/{(?!{)([\w\d]+)}(?!})/)
         if (m !== null){
             var left = s.substr(0, m.index)
-            var sep = s.substr(m.index, m[0].length)
+            //var sep = s.substr(m.index, m[0].length)
             s = s.substr(m.index+m[0].length)
             var n = parseInt(m[1])
             splits.push(left)
-            if (n != n){ // not a number
+            if (n !== n){ // not a number
```

Comment:
I think this line intentionally uses type conversion via`!=` instead of the strict `!==`.